marshmallow/schema.py

```diff
@@ -19,7 +22,31 @@
 from marshmallow.orderedset import OrderedSet
 from marshmallow.decorators import (PRE_DUMP, POST_DUMP, PRE_LOAD, POST_LOAD,
                                     VALIDATES, VALIDATES_SCHEMA)
-from marshmallow.utils import missing
+from marshmallow.utils import missing, suppress
+
+DEFAULT_JIT_ENVIRONMENT_VARIABLE = 'MARSHMALLOW_SCHEMA_DEFAULT_JIT'
```

Comment:
I don't think this configuration parsing belongs in marshmallow. As I suggested in my previous comment, users can do this easily in their application code

```python
BaseSchema = JITSchema if os.environ.get('USE_JIT') else Schema
JITClass = import_object(os.environ.get('MARSHMALLOW_SCHEMA_DEFAULT_JIT'))
```